doc/api/readline.md

```diff
@@ -369,7 +369,7 @@ changes:
   * `crlfDelay` {number} If the delay between `\r` and `\n` exceeds
     `crlfDelay` milliseconds, both `\r` and `\n` will be treated as separate
     end-of-line input. Default to `100` milliseconds.
-    `crlfDelay` will be coerced to `[100, 2000]` range.
+    `crlfDelay` will be coerced to `100` or greater, `Infinity` is allowed.
```

Comment:
Hmm, this could be clearer.

Maybe something along the lines of:
```
`crlfDelay` will be coerced to an Integer no less than `100`, unless it is `Infinity`.
```